.env

```diff
@@ -1,3 +1,3 @@
-REACT_APP_DISCOVER=chance-source-liquid-nelson
-REACT_APP_VERSION=$npm_package_version
-REACT_APP_ENVIRONMENT=prod
+VITE_DISCOVER=chance-source-liquid-nelson
+VITE_VERSION=$npm_package_version
+VITE_ENVIRONMENT=prod
```

Comment:
I think there is a built in for this called mode

Reply:
That's correct and I wasn't using it anyways.